src/core/dev/net_device_val.cpp

```diff
@@ -734,21 +734,18 @@ void net_device_val::verify_bonding_mode()
     }
 
     if (m_bond == NO_BOND || m_bond_fail_over_mac > 1) {
-        vlog_printf(
+        __log_raw(
             VLOG_WARNING,
             "******************************************************************************\n");
-        vlog_printf(VLOG_WARNING,
-                    "" PRODUCT_NAME " doesn't support current bonding configuration of %s.\n",
-                    get_ifname_link());
-        vlog_printf(
+        __log_warn("" PRODUCT_NAME " doesn't support current bonding configuration of %s.\n",
+                   get_ifname_link());
+        __log_raw(
```

Comment:
log_warn?

Reply:
regex search and replace wasn't good enough - thanks :) fixed more occurences as well.